tests/scripts/generate_test_code.py

```diff
@@ -126,9 +126,9 @@
 This script replaces following fields in the template and generates
 the test source file:
 
-$test_common_helpers        <-- All common code from helpers.function
+/*%test_common_helpers%*/   <-- All common code from helpers.function
```

Comment:
Why not use valid C identifier as markers? This should avoid confusing any tool that parses C, and for tools that work after preprocessing, we could potentially `-D` those identifiers.